CHANGELOG.md

```diff
@@ -2,17 +2,25 @@
 
 ## 0.0.11
 
-- BREAKING CHANGES: Keep a backup of your VIN & MAC Addr
+### Breaking change & Upgrade Instruction
+- No need to provide MAC addresses anymore for presence detection
```

Comment:
Not true just yet. No need to provide MAC address IF the end user doesn't want to do presence detection.

Reply:
Yes indeed